docs/alexa-directives.rst

```diff
@@ -95,6 +95,121 @@ Voxa provides a `DisplayTemplate` builder that can be used with the `alexaRender
     });
 
 
+Alexa Presentation Language (APL) Templates
+--------------
+
+`Alexa Documentation <https://developer.amazon.com/docs/alexa-presentation-language/apl-overview.html>`_
+
+An APL Template is sent with the `alexaAPLTemplate` key in your controller. You can pass the directive object directly or a view name with the directive object.
+
+One important thing to know is that is you sent a Render Template and a APL Template in the same response but the APL Template will be the one being rendered if the device supports it; if not, the Render Template will be one being rendered.
+
+.. code-block:: javascript
+
+  // variables.js
+
+    exports.MyAPLTemplate = (voxaEvent) => {
+      // Do something with the voxaEvent, or not...
+
+      return {
+        datasources: {},
+        document: {},
+        token: "SkillTemplateToken",
+        type: "Alexa.Presentation.APL.RenderDocument",
+      };
+    });
+
+  // views.js
+
+    const views = {
+      "en-US": {
+        translation: {
+          MyAPLTemplate: "{MyAPLTemplate}"
+        },
+      };
+    };
+
+  // state.js
+
+    app.onState('someState', () => {
+      return {
+        alexaAPLTemplate: "MyAPLTemplate",
+      };
+    });
+
+    // Or you can do it directly...
+
+    app.onState('someState', () => {
+      return {
+        alexaAPLTemplate: {
+          datasources: {},
+          document: {},
+          token: "SkillTemplateToken",
+          type: "Alexa.Presentation.APL.RenderDocument",
+        },
+      };
+    });
+
+
+Alexa Presentation Language (APL) Commands
```

Comment:
would be great to also mention the XML tool we use to build this stuff

Reply:
Let's keep it separate for now, if just so we can work on it separately from the main voxa repo. But let's also make it a public one. For now, i'm approving this PR